scipy-1.0/paper.tex

```diff
@@ -122,6 +122,21 @@ \section*{Project organisation and community}
 
 \textbf{Governance}
 
+SciPy adopted an official Governance Document on August 3, 2017 (see PR\#6953). In summary,
+we have a Benevolant Dictator for Life (BDFL) in Pauli Virtanen, and a (currently) 18
+member Steering Council chaired by Ralf Gommers that i.e., decides on who gets commit rights.
+
+SciPy adopted an official Code of Conduct on October 24, 2017 (see PR\#7963), drawing
+inspiration from the Apache Foundation Code of Conduct, the Contributor Covenant,
+the Jupyter Code of Conduct, and the Open Source Guides - Code of Conduct. In short,
+we stated five specific guidelines: \emph{be open} (to anyone participating in our community),
+\emph{be empathetic and patient} (in resolving conflicts), \emph{be collaborative} (we depend
+on each other to build up the tools in the library), \emph{be inquisitive} (identifying issues
+early on can be helpful to everyone), and \emph{be careful with wording} (do not harass or exclude).
+We outline a broad diversity inclusion statement, and provide
```

Comment:
The tense has changed from "we stated" to "we outline". I would suggest past tense if we are talking about what we did, or better might be to state in present tense what the document does e.g. "it contains a broad diversity... and provides..."